srcrepo.go

```diff
@@ -107,7 +107,7 @@ func (m *Manager) installSrc(version *GoVersion) (string, error) {
 
 	// move final build result into destination
 	log.Println("rename")
-	if err = os.Rename(filepath.Join(tmpRoot, "go"), to); err != nil {
+	if err = copyDirectory(filepath.Join(tmpRoot, "go"), to); err != nil {
```

Comment:
Every place that has been switched over to use `copyDirectory` now needs to delete the source directory after the copy completes.